updatecli-matrix-targets.yaml

```diff
@@ -13,21 +13,21 @@ matrix:
       compose_key: $.services.alfresco.image
       helm_target: &helmvalues231 >-
         helm/alfresco-content-services/values.yaml
-      helm_key: $.repository.image.tag
+      helm_key: .repository.image.tag
```

Comment:
why? updatecli complains without leading `$`

Reply:
I just valiate that this is ue to the target yaml node having an anchor